modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -129,7 +129,36 @@ ifdef::ipi[]
 . Record the image details of your offer. You must update the `compute` section in the `install-config.yaml` file with values for `publisher`, `offer`, `sku`, and `version` before deploying the cluster.
 endif::ipi[]
 ifdef::upi[]
-. Record the image details of your offer. If you use the Azure Resource Manager (ARM) template to deploy your worker nodes, you can update `storageProfile.imageReference` by deleting the `id` parameter and adding the `offer`, `publisher`, `sku`, and `version` parameters using the values from your offer.
+. Record the image details of your offer. If you use the Azure Resource Manager (ARM) template to deploy your worker nodes:
++
+.. Update `storageProfile.imageReference` by deleting the `id` parameter and adding the `offer`, `publisher`, `sku`, and `version` parameters by using the values from your offer.
+.. Specify a `plan` for the virtual machines (VMs). 
++
+.Sample `06_workers.json` ARM template with an updated `storageProfile.imageReference` object and a specified `plan` 
++
+[source,json]
+----
+...
+  "storageProfile": {
+    "imageReference": {
+      "offer": "rh-ocp-worker",
+      "publisher": "redhat",
+      "sku": "rh-ocp-worker",
+      "version": "4.8.2021122100"
+  }
+}
+...
+  "plan" : {
+    "name": "rh-ocp-worker",
+    "product": "rh-ocp-worker",
+    "publisher": "redhat-limited"
+  },
+  "dependsOn" : [
+    "[concat('Microsoft.Network/networkInterfaces/', concat(variables('vmNames')[copyIndex()], '-nic'))]"
+  ],
+...
```

Comment:
Checked 06_worker.json file, better to move "plan"..."dependsOn"  in front of storageProfile to easily find the place to update.

Reply:
Thanks. I moved this section. Does the `],` syntax still stand on line 149 (on updated file)?

Reply:
no, I checked 06_worker.json, it should be correct.

seems 'storageProfille' is not at the same level with "plan" and "dependsOn", suggest to check the content of 06_worker.json in section "Creating additional worker machines in Azure" and adjust the format here.